src/cljc/puumerkki/codec.cljc

```diff
@@ -1,27 +1,13 @@
 (ns puumerkki.codec
-  #?(:cljs (:require [goog.crypt :as crypt])))
+  (:refer-clojure :exclude [abs bigint])
+  (:require [puumerkki.bigint :as bigint :refer [bigint bigint?]]
+            #?(:cljs [goog.crypt :as crypt])))
 
 ;;; misc utils
 
-(defn fold [op st lst]
-  (if (empty? lst)
-    st
-    (recur op (op st (first lst)) (rest lst))))
-
-;; pred lst → bool
-(defn all [op lst]
-  (cond
-    (empty? lst) true
-    (op (first lst))
-    (recur op (rest lst))
-    :true false))
-
-(defn zip [op la lb]
-  (loop [la la lb lb out ()]
-    (if (or (empty? la) (empty? lb))
-      (reverse out)
-      (recur (rest la) (rest lb)
-             (cons (op (first la) (first lb)) out)))))
+(def max-safe-integer-bytes
+  #?(:cljs 4
+     :clj 7))
 
 (defn abs [n]
   (if (< n 0) (* n -1) n))
```

Comment:
Sivuhuomio (ei ole suoraan tämään PR:n asiaa): 
Voisi samalla varmistaa, että käytetään clojuren funktioita, `neg?`, `zero?` ja `pos?`. Näitä näytti olevan muutamassa kohdassa tässä namespacessa.

Reply:
Jep, koodissa on aika paljon kohtia mitä voisi korvata korkeamman tason clojure.core-funktioilla, mutta en nyt lähtenyt siihen. Itseasiassa koko abs-funktion voisi korvata uusimman coren vastaavalla.